### PR TITLE
test: FAQ module tests

### LINKED ISSUE #678: test: Create tests for FAQ list module
**Is it a ?**

- [ ] Bug Report

- [ ] Feature Request
    
- [ ] Chore
    
**Description**

Create unit tests for FAQ module.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #678 

Changes: 
1. Added tests for FaqPresenter.
2. Added tests for FaqRepository.

Screenshots for the change: 
Not applicable.

### CODE REVIEW

**Comment by Masquerade0097** [comment]
@iamareebjamal Please review.

File: app/src/test/java/org/fossasia/openevent/app/unit/model/FaqRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.fossasia.openevent.app.unit.model;
+
+/**
+ * Created by priyanshu on 2/3/18.
+ */
+
+public class FaqRepositoryTest {
+}
```
Comment (iamareebjamal) [inline app/src/test/java/org/fossasia/openevent/app/unit/model/FaqRepositoryTest.java]: This file is empty

Comment (Masquerade0097) [reply app/src/test/java/org/fossasia/openevent/app/unit/model/FaqRepositoryTest.java]: Sorry for that. I've updated the PR, Please have a look

Comment (Masquerade0097) [reply app/src/test/java/org/fossasia/openevent/app/unit/model/FaqRepositoryTest.java]: Sorry for that. I've updated the PR, please have a look.

**Pull request merged**